client/client-multi/client-cpp/impl/configuration/storage/FileConfigurationStorage.cpp

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2014-2015 CyberVision, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "kaa/configuration/storage/FileConfigurationStorage.hpp"
+
+#include <fstream>
+#include <iterator>
+
+namespace kaa {
+
+
+void FileConfigurationStorage::saveConfiguration(std::vector<std::uint8_t>&& bytes)
+{
+    std::ofstream outFile(filename_, std::ofstream::binary);
+    if (outFile.good()) {
```

Comment:
We could simplify it by writing  just  'if (outFile)'  instead of 'if (outFile.good())'